deliverables/README.md

```diff
@@ -12,16 +15,37 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
+In an interpreted language, the code we write is translated and run through an interpreter which performs the user's desired actions.
+
 What is IRB?<br>
+IRB stands for interactive Ruby. It interactively executes whatever Ruby code we write one line at a time.
+
 What are Objects?<br>
+Every thing in Ruby is an object. Objects are classified by the characteristics which make up a class of objects. In addition to an objects defining characteristics, you can perform functions/methods on objects.
+
 What are Methods?<br>
+Methods, or functions. are a set of instructions bound to an object and have access to its instance variables. You call methods on objects.
+
 What are Classes?<br>
+A class is the blueprint for which objects are created. Classes classify objects and each class of object has certain characteristics and functions associated to them.
+
 What are Blocks?<br>
+Blocks in Ruby are sets of instructions, placed inside curly braces, which can take arguments and return values and are invoked using a yield statement. They are syntax literals for Proc objects.
+
 What is an Array?<br>
+An array is a list of stored information. The list in an array has order. 
+
```

Comment:
Arrays elements dont have to be ordered. They are however 0 indexed. 

     array = [   1,    7,     4,      a]
                     ^0    1      2       3  # this is the index of each item in the array